bundles/org.openhab.binding.airq/README.md

```diff
@@ -0,0 +1,229 @@
+# airq Binding
+
+The airq Binding integrates the air analyzer <a href="http://www.air-q.com">air-Q</a> device into the openHAB system.
+
+With the binding, it is possible to subscribe to all data delivered by the air-Q device.
+
+<img src="https://uploads-ssl.webflow.com/5bd9feee2fb42232fe1d0196/5e4a8dc0e322ca33891b51e4_air-Q%20frontal-p-800.png" alt="air-Q image" width="400px" height="324px" />
+
+## Supported Things
+
+One only Thing is supported: the air-Q device.
+This Binding was tested with an air-Q Pro device with 14 sensors. It should also work with an air-Q device with 11 sensors, but it was not tested yet.
+
+## Discovery
+
+Auto-discovery is not possible in this version. Since the binding has to be configured at least with the password of the device, auto-discovery would be of limited value anyway.
+
+## Binding Configuration
+
+The binding does not need to be configured.
+
+## Thing Configuration
+
+The air-Q thing must be configured with (both mandatory):
+| Parameter | Description                        |
+|-----------|------------------------------------|
+| ipAddress | Network address, e.g. 192.168.0.68 |
+| password  | Password of the air-Q device       |
+
+The Thing provides the following properties:
+| Parameter              | Description                   |
+|------------------------|-------------------------------|
+| id                     | Device ID                     |
+| air-Q-Hardware-Version | Hardware version              |
+| air-Q-Software-Version | Firmware version              |
+| sensors                | Available sensors             |
+| SensorInfo             | Information about the sensors |
+| Industry               | Industry version              |
```

Comment:
Parameters should be camelCase, so consider changing them to something like this:

SensorInfo to sensorInfo
Industry  to industryVersion
air-Q-Hardware-Version to hardwareVersion